source/includes/_build.md

```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
 
 ## Linux or macOS
 
-For Iroha, a provisioning of Postgres and Redis are required. You may launch them on your local machine, or use docker containers, as provided.
-
 > Launching Docker and Postgres in Docker
 
 ``` shell
-docker run --name some-redis -p 6379:6379 -d redis:3.2.8
-docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres:9.5
+docker run --name some-redis \
+-p 6379:6379 -d redis:3.2.8
+docker run --name some-postgres \
+-e POSTGRES_USER=iroha \
+-e POSTGRES_PASSWORD=helloworld \
+-p 5432:5432 -d postgres:9.5
 ```
 
-> Linux
+To launch Iroha daemon, running postgres and redis services are required. You may launch them on your local machine, or use docker containers, as provided on the right side.
 
-``` shell
-apt-get -y --no-install-recommends install \
-        build-essential python-software-properties \
-        automake libtool \
-        libssl-dev zlib1g-dev libboost-all-dev \
-        libc6-dbg golang \
-        git ssh tar gzip ca-certificates \
-        python3 python3-pip python3-setuptools \
-        lcov \
-        wget curl cmake file unzip gdb \
-        iputils-ping vim ccache \
-        gcovr vera++ cppcheck doxygen \
-        graphviz graphviz-dev; \
-    apt-get -y clean
-git clone https://github.com/hyperledger/iroha -b develop
-cd iroha
-cmake -H. -Bbuild
-cmake --build build -- -j4
-```
+### Linux (debian-based)
 
-> macOS
+To install dependencies, clone, and build the project, please use this code:
+<script src="https://gist.github.com/neewy/39557aa444c3317aeffbdedd9f0f51e2.js"></script>
+
+#### Outdated dependencies 
+
+In case that some dependencies are outdated in apt, you are advised to use following sources to install dependencies:
+
+##### Boost
+To install Boost libraries (`libboost-all-dev`), use [current release](http://www.boost.org/users/download/) from Boost webpage.
+Exact dependencies are system and filesystem, so use `./bootstrap.sh --with-libraries=system,filesystem` when you are building the project.
+
+##### CMake
+To install CMake tool (`cmake`), use [latest release](https://cmake.org/download/) from CMake webpage.
+
+### macOS
 
-``` shell
-xcode-select --install
-# if you dont't have brew installed
-ruby -e "$(curl -fsSL https://raw.githubusercontent.com/homebrew/install/master/install)"
-brew install cmake boost postgres grpc autoconf automake libtool golang
-git clone https://github.com/hyperledger/iroha -b develop
-cd iroha
-cmake -H. -Bbuild
-cmake --build build -- -j4
-```
 > macOS brew
 
 ``` shell
 brew tap soramitsu/iroha
 brew install iroha
 ```
+
+To install dependencies, clone, and build the project, please use this code:
+<script src="https://gist.github.com/neewy/bc0fea777592a5381aa4ab6e68bfe699.js"></script>
```

Comment:
Same issue with <script> tag.

Reply:
Is there a way to see the code inside js file you are pointing here?

Reply:
yes, it shows fine

Reply:
<img width="741" alt="screen shot 2017-12-16 at 16 50 14" src="https://user-images.githubusercontent.com/11841667/34071073-9e07865e-e281-11e7-90cb-666d7fb78f70.png">